doc/source/quickstart/quickstart-hologres-with-tableau.md

```diff
@@ -0,0 +1,849 @@
+  LateAircraftDelay numeric(13,2),
+  FirstDepTime text,
+  TotalAddGTime numeric(13,2),
+  LongestAddGTime numeric(13,2),
+  DivAirportLandings numeric(13,2),
+  DivReachedDest numeric(13,2),
+  DivActualElapsedTime numeric(13,2),
+  DivArrDelay numeric(13,2),
+  DivDistance numeric(13,2),
+  Div1Airport text,
+  Div1AirportID integer,
+  Div1AirportSeqID integer,
+  Div1WheelsOn text,
+  Div1TotalGTime numeric(13,2),
+  Div1LongestGTime numeric(13,2),
+  Div1WheelsOff text,
+  Div1TailNum text,
+  Div2Airport text,
+  Div2AirportID integer,
+  Div2AirportSeqID integer,
+  Div2WheelsOn text,
+  Div2TotalGTime numeric(13,2),
+  Div2LongestGTime numeric(13,2),
+  Div2WheelsOff text,
+  Div2TailNum text,
+  Div3Airport text,
+  Div3AirportID integer,
+  Div3AirportSeqID integer,
+  Div3WheelsOn text,
+  Div3TotalGTime numeric(13,2),
+  Div3LongestGTime numeric(13,2),
+  Div3WheelsOff text,
+  Div3TailNum text,
+  Div4Airport text,
+  Div4AirportID integer,
+  Div4AirportSeqID integer,
+  Div4WheelsOn text,
+  Div4TotalGTime numeric(13,2),
+  Div4LongestGTime numeric(13,2),
+  Div4WheelsOff text,
+  Div4TailNum text,
+  Div5Airport text,
+  Div5AirportID integer,
+  Div5AirportSeqID integer,
+  Div5WheelsOn text,
+  Div5TotalGTime numeric(13,2),
+  Div5LongestGTime numeric(13,2),
+  Div5WheelsOff text,
+  Div5TailNum text,
+  trailer integer
+);
+
+CALL set_table_property('ontime', 'clustering_key', 'FltYear, FltQuarter, FltMonth');
+CALL set_table_property('ontime', 'segment_key', 'FltYear');
+CALL set_table_property('ontime', 'shard_count', '1');
+CALL set_table_property('ontime', 'bitmap_columns', 'OriginState, DestState, FltDayofMonth, FltDayOfWeek, Carrier');
+CALL set_table_property('ontime', 'dictionary_encoding_columns', 'UniqueCarrier, Carrier, TailNum, Origin, OriginCityName, OriginState, OriginStateName, Dest, DestCityName, DestState, DestStateName, DepTimeBlk, WheelsOff, WheelsOn, ArrTimeBlk, CancellationCode, Div1Airport, Div1WheelsOn, Div1WheelsOff, Div1TailNum, Div2Airport, Div2WheelsOn, Div2WheelsOff, Div2TailNum, Div3Airport, Div3WheelsOn, Div3WheelsOff, Div3TailNum, Div4Airport, Div4WheelsOn, Div4WheelsOff, Div4TailNum, Div5Airport, Div5WheelsOn, Div5WheelsOff, Div5TailNum');
+CALL set_table_property('ontime', 'time_to_live_in_seconds', '31536000');
+COMMIT;
+```
+
+After you finish the script file, execute the script file.
+
+```
+PGPASSWORD=<password> psql -U <username> -h <ip> -p <port> -d <database> -f /temp/ddl/data.ddl
+```
+
+```
+BEGIN
+CREATE TABLE
+CALL
+CALL
+CALL
+CALL
+CALL
+CALL
+CALL
+COMMIT
+```
+
+Let's check the table "ontime" we have executed.
+
+```
+\d ontime
+```
+
+Now you have prepared one database with one table.
+
+## Copy Data into Hologres 
+
+Let's copy data into table (374 files in total). Note: multiple csv files should be uploaded in 5 minutes by the following script:
+
+```
+cat /temp/*.csv |PGPASSWORD=[] psql -U [user] -h [host] -p [port] -d [database] -c 'copy ontime from stdin CSV HEADER'
```

Comment:
I might have made a mistake here. I think your original approach with "\copy ontime FROM '/temp/testdata_file/On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_1987_10.csv' DELIMITER ',' CSV HEADER;" should be good, and we should stick to this approach